### DISCUSSION TITLE
Change alpha to latest

### DISCUSSION THREAD

**Discussion by YasserKhalilExcelEgy** [pyscript/pyscript#1491] [Q&A] [answered]
I have this working pyscript code using `alpha` in the head section. When trying to change to `latest` I encountered an error

````
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF8">
		<title>PyScript Hello, World!</title>
		<link rel="stylesheet" href="https://pyscript.net/latest/pyscript.css" />
		<script defer src="https://pyscript.net/latest/pyscript.js"></script>
		<py-env>
            - numpy
            - matplotlib
		</py-env>
	</head>
	<body>
		<py-script>
			import numpy as np
			import matplotlib.pyplot as plt

			print('Hello World')

			a = 0
			b = 1
			for _ in range(10):
				print(a)
				a, b = b, a + b

			x, y = np.random.random(100), np.random.random(100)
			plt.scatter(x, y)
			plt.gcf()
		</py-script>
	</body>
</html>
````

The error looks like that
``
Traceback (most recent call last):
  File "/lib/python3.10/_pyodide/_base.py", line 460, in eval_code
    .run(globals, locals)
  File "/lib/python3.10/_pyodide/_base.py", line 306, in run
    coroutine = eval(self.code, globals, locals)
  File "<exec>", line 1, in <module>
ModuleNotFoundError: The module 'numpy' is included in the Pyodide distribution, but it is not installed.
You can install it by calling:
  await micropip.install("numpy") in Python, or
  await pyodide.loadPackage("numpy") in JavaScript
See https://pyodide.org/en/stable/usage/loading-packages.html for more details.
``

**Comment by madhur-tandon** [comment]
Hi @YasserKhalilExcelEgy 

The `py-env` tag is no longer supported. The correct tag to use is `py-config` and you can refer here: https://docs.pyscript.net/latest/reference/elements/py-config.html#dependencies-and-packages for more information.

**Reply by madhur-tandon** [reply, marked as answer]
In particular, you can use the following snippet:

```
<py-config>
    packages = ["numpy", "matplotlib"]
</py-config>
```

**Reply by YasserKhalilExcelEgy** [reply]
Thanks a lot. Now no errors but the plot not displayed as was in alpha mode.

**Reply by madhur-tandon** [reply]
You can do `display(plt)` as mentioned here: https://docs.pyscript.net/latest/reference/API/display.html#using-matplotlib-with-display

**Reply by YasserKhalilExcelEgy** [reply]
Great. Now it worked. I just need to display it at the end of the page.
